includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -463,6 +476,7 @@ public function wc_setup_store_setup_save() {
 		$postcode       = sanitize_text_field( $_POST['store_postcode'] );
 		$currency_code  = sanitize_text_field( $_POST['currency_code'] );
 		$product_type   = sanitize_text_field( $_POST['product_type'] );
+		$sell_in_person = 'yes' === sanitize_text_field( $_POST['sell_in_person'] );
```

Comment:
This appears to trigger an "Undefined index" notice in the not-checked case, since the param is just not set.